Makefile

```diff
@@ -80,8 +86,8 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 .PHONY: test
-test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out
+test: manifests kustomize generate fmt vet envtest ## Run tests.
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out -v $(GINKGO_VERBOSE_FLAG) $(GINKGO_LABELS_FLAG)
```

Comment:
Do not pass values here, cause testing via IDE do not take it into account. Support verbose and label in the siute_test.go istelf. Then in the suite_test handle reading those envs.

Reply:
Move KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) to the set-env-vars.sh. That will make IDE testing possible too.

Reply:
Done, but it means the risk if LOCALBIN and ENVTEST changes in Makefile without syncing with the script, unless we pass these values to the script and this seems awkward - passing envs from make to script and from script to make back.

Reply:
> Move KUBEBUILDER_ASSETS="$(shell 
> (
> E
> N
> V
> T
> E
> S
> T
> )
> u
> s
> e
> (ENVTEST_K8S_VERSION) to the set-env-vars.sh. That will make IDE testing possible too.

And we create dependency in set-env-variables.sh (definition of the current version of kubebuilder)

Reply:
Changes pushed